src/main/java/thePackmaster/cards/sneckopack/SneckoEyes.java

```diff
@@ -0,0 +1,29 @@
+package thePackmaster.cards.sneckopack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.ConfusionPower;
+import com.megacrit.cardcrawl.powers.DrawPower;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class SneckoEyes extends AbstractPackmasterCard {
+
+
+    public final static String ID = makeID("SneckoEyes");
+
+    public SneckoEyes() {
+        super(ID, 3, CardType.POWER, CardRarity.RARE, CardTarget.SELF);
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        addToBot(new ApplyPowerAction(p,p, new ConfusionPower(p)));
```

Comment:
You think we should also randomize the cost of current cards in hand?

Reply:
I don't think so, no. It would crowd the card text, and tbh I don't like powers that do something that is Skill-like along with their actual effect.